test/functional/test_serial_number.py

```diff
@@ -0,0 +1,107 @@
+'''
+*********************************************************
+Copyright @ 2018 Dell EMC Corporation All Rights Reserved
+*********************************************************
+'''
+import unittest
+import os
+from test import fixtures
+from infrasim import model
+from infrasim import helper
+
+
+conf = None
+OLD_PATH = os.environ.get("PATH")
+NEW_PATH = "{}/bin:{}".format(os.environ.get("PYTHONPATH"), OLD_PATH)
+ssh = None
+
+
+def setup_module():
+    os.environ["PATH"] = NEW_PATH
+
+
+def teardown_module():
+    if conf:
+        stop_node()
+    os.environ["PATH"] = OLD_PATH
+
+
+def start_node():
+    """
```

Comment:
better to remove these comments here, seems they have no relationship with this serial number case.

Reply:
ok .thanks